File: MyApplication/app/src/main/java/com/example/myapplication/MainActivity.java
package com.example.myapplication;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import android.util.Log;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    TextView numberOne;
    TextView numberTwo;

    TextView novo;

    Button sum;

    TextView result1;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        numberOne = findViewById(R.id.numberOne);
        numberTwo = findViewById(R.id.numberTwo);

        result1 = findViewById(R.id.result1);

        sum = findViewById(R.id.button);
        sum.setOnClickListener(new View.OnClickListener() {
            public void onClick(View v) {
                int one = Integer.parseInt(numberOne.getText().toString());
                int two = Integer.parseInt(numberTwo.getText().toString());
                int result = one + two;
                result1.setText("result: " + result);
            }
        });
    }

}

File: sum/app/src/main/java/com/example/sum/MainActivity.kt
package com.example.sum

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.EditText
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        var _num1 = findViewById<EditText>(R.id.valor1)
        var _num2 = findViewById<EditText>(R.id.valor2)

        btnSoma.setOnClickListener(){
            var num1 = _num1.text.toString().toInt()
            var num2 = _num2.text.toString().toInt()
            valorResultado.text = "$num1 + $num2 = " + (num1 + num2).toString()

        }
    }
}
